pkg/version/version.go

```diff
@@ -0,0 +1,4 @@
+package version
+
+// Version is the version of the build.
+const Version = "0.14.1-dev"
```

Comment:
Can you bump this back to `0.15.0-dev`. Apologies for the back and forth. But as we have a branch for 0.14, the next release from master will be 0.15. Having that here will prevent us from accidentally releasing a wrong version.

Reply:
Yes sure, adapted as suggested. 🙂